Makefile

```diff
@@ -0,0 +1,61 @@
+####### Universal ######
+
+pip:
+	pip install --upgrade pip
```

Comment:
I'd rather not have build tool versions constantly shifting under our feet

Reply:
👍

Reply:
👍